modules/core/02-client/abci.go

```diff
@@ -26,7 +26,8 @@ func BeginBlocker(ctx sdk.Context, k keeper.Keeper) {
 			}
 			bz := k.MustMarshalConsensusState(upgradedConsState)
 
-			k.SetUpgradedConsensusState(ctx, plan.Height, bz)
+			k.SetUpgradedConsensusState(ctx, plan.Height, bz) //nolint:errcheck // this could make the chain halt if we checked the error.
```

Comment:
```suggestion
			k.SetUpgradedConsensusState(ctx, plan.Height, bz) //nolint:errcheck 
```
Is there a way to silence the code scanning alert? In this situation [`SetUpgradedConsensusState` never returns an error](https://github.com/cosmos/cosmos-sdk/blob/main/x/upgrade/keeper/keeper.go#L229). It is just by mistake I believe that the API includes a returned error